runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -2579,7 +2564,6 @@ remoteCompile(
          auto comp = compInfoPT->getCompilation();
          if (TR::Options::getVerboseOption(TR_VerboseCompilationDispatch))
             TR_VerboseLog::writeLineLocked(TR_Vlog_DISPATCH, "Interrupting remote compilation of %s @ %s", comp->signature(), comp->getHotnessName());
-         client->writeError();
```

Comment:
Like in other cases, we need a specific message to let the server know the client is not going to continue with this compilation.